tests/sentry/receivers/test_superuser.py

```diff
@@ -0,0 +1,53 @@
+from sentry.auth.superuser import is_active_superuser
+from sentry.models import User
+from sentry.receivers.superuser import disable_superuser, enable_superuser
+from sentry.testutils import TestCase
+
+
+class SuperuserReceiverTest(TestCase):
```

Comment:
Lets also add some negative tests here

Reply:
I'm not too sure what negative tests could be written. The superuser receiver has the following cases for each of the methods. 

`enable_superuser`: 
- superuser self-hosted: superuser active
- superuser flag on: superuser active
- superuser saas settings: do nothing
- not superuser self-hosted: do nothing
- not superuser flag on:  do nothing
- not superuser saas settings: do nothing

`disable_superuser`: 
- superuser: disable superuser
- non superuser: do nothing

I did miss the base case so I've added that but I believe I've covered the cases above now